packages/block-editor/src/components/block-navigation/tree.js

```diff
@@ -21,11 +21,18 @@ import { BlockNavigationContext } from './context';
  */
 export default function BlockNavigationTree( {
 	__experimentalWithBlockNavigationSlots,
+	__experimentalWithEllipsisMenu,
 	...props
 } ) {
 	const contextValue = useMemo(
-		() => ( { __experimentalWithBlockNavigationSlots } ),
-		[ __experimentalWithBlockNavigationSlots ]
+		() => ( {
+			__experimentalWithBlockNavigationSlots,
+			__experimentalWithEllipsisMenu,
+		} ),
+		[
+			__experimentalWithBlockNavigationSlots,
+			__experimentalWithEllipsisMenu,
+		]
```

Comment:
Why are we using context for these two settings? Would prop-passing be simpler? 🙂

Reply:
Normally I'd say it would be, but this is quite tangled network of components structured like a tree - we'd have to pass these two props through ~5 layers of components. Context is indeed much simpler here.

Reply:
I also think context saving some repeated props works.